interface/forms/requisition/new.php

```diff
@@ -0,0 +1,368 @@
+       padding: 5px;
+	   position: relative;
+	   float: left;
+	   width: 255px;
+	   height: 125px;
+   }
+ 
+  .dx {
+      padding: 5px;
+	  position: relative;
+	  float: right;
+	  border-style: solid;
+	  border-width: 1px;
+	  width: 130px;
+	  height: 125px;
+  }
+  
+  .plist {
+      padding: 5px;
+	  position: relative;
+	  float: left;
+	
+  }
+  
+  .pFill {
+	  float: left;
+	  
+  }
+  .barcode{
+	  position: relative;
+	  right: -380px;
+  }
+</style>
+</head>
+
+<body>
+<div class="container">
+        <div class="barcode">
+        <br>
+        <br>
+            <?php
+            /**
+             *  This is to store the requisition bar code number to use again if the form needs to be printed or viewed again
+             *  But save it the first time through.
+             */
+               $lab_id = $order[0];
+               $storeBar = getBarId($lab_id,$pid);
+
+            if (!empty($storeBar)) {
+                $bar = $storeBar['req_id'];
+            } else {
+                $bar = rand(1000,999999);
+                saveBarCode($bar,$pid,$order[0]);
+            }
+
+            ?>
+            <img  src="../../forms/requisition/barcode.php?text=<?php echo $bar; ?>" alt="barcode" /></br>
+        &#160;&#160;&#160;&#160;&#160;	<?php echo text($bar); ?>
+        </div>
+        <div class="reqHeader" id="printableArea">
+        <p><font size="4"><b>Req #:</b> <?php echo text($bar); ?>  &#160;&#160;&#160;&#160;&#160;&#160;<b>Client #:</b> <?php echo text($provLabId['recv_fac_id']); ?></font></p>
+           <div class="cinfo">
+           <font size="4">
+               <?php echo xlt($facility['name']) ."<br>". xlt($facility['street']) . "<br>" .
+                          xlt($facility['city']).",".xlt($facility['state']).",".text($facility['postal_code']) . "<br>" .
+                          text($facility['phone']); ?>
+                          </font>
+           </div>
+           <div class="pdata">
+                 <p><font size="4">
+          <?php echo xlt($pp['organization'])."</br>".
+          xlt($pp['street'])." | ".xlt($pp['city']).", ".xlt($pp['state'])." ".text($pp['zip'])."</br>".
+          "O:".text($pp['phone'])." | F:".text($pp['fax'])."</br>";
+                 ?></font></p>
+
+           </div>
+        </div>
+        <div class="req" id="printableArea">
+            <table class="table" style="width:800px" border="1">
+               <tr style="height:125px;">
+                   <td style="vertical-align:top; width:400px;" >
+                   <div class="plist">
+                       <?php echo xlt('Collection Date/Time')?>:</br>
+                       <?php echo xlt('Lab Reference ID') ?>:</br>
+                       <?php echo xlt('Fasting')?>:</br>
+                       <?php echo xlt('Hours')?>:</br>
+                     </div>
+                    <div class="pFill">
+                      <?php echo $order[6];?> </br>
+                        <?php echo $order[0];?>
```

Comment:
What are examples of $order[6] and $order[0] (to see if these need html escaping via text() function)

Reply:
It is text being echoed to the screen and should be wrapped.